scripts/update_teams.py

```diff
@@ -39,10 +41,10 @@
 
 class NullUndefined(jinja2.Undefined):
     def __unicode__(self):
-        return unicode(self._undefined_name)
+        return six.text_type(self._undefined_name)
 
     def __getattr__(self, name):
-        return unicode('{}.{}'.format(self, name))
+        return ('{}.{}'.format(self, name))
```

Comment:
Or actually I'm not sure.  This is asymmetric with the same class in watch_only_my_feedstocks.py.  Probably best to just make sure both are unicode in python 2